in-app-wallet/typescript/src/in_app_wallet_examples.ts

```diff
@@ -0,0 +1,157 @@
+import { 
+    Connection, 
+    JsonRpcProvider, 
+    TransactionBlock,
+    SuiTransactionBlockResponse
+} from "@mysten/sui.js";
+import { rpcClient } from "typed-rpc";
+
+// The Key Service is Shinami's secure and stateless way to get access to the In-App Wallet
+const KEY_SERVICE_RPC_URL = "https://api.shinami.com/key/v1/<API_ACCESS_KEY>";
+
+// The Wallet Service is the endpoint to issue calls on behalf of the wallet.
+const WALLET_SERVICE_RPC_URL = "https://api.shinami.com/wallet/v1/<API_ACCESS_KEY>";
+
+// Shinami Sui Node endpoint + Mysten provided faucet endpoint:
+const connection = new Connection({
+    fullnode: 'https://node.shinami.com/api/v1/<API_ACCESS_KEY>',
+    faucet: 'https://faucet.testnet.sui.io/gas'
+});
+
+const suiProvider = new JsonRpcProvider(connection);
+const walletId = "<WALLET_ID>";
+const secret = "<SECRET>";
+const GAS_BUDGET = 5000000;
+
+// Key Service interaction setup
+interface KeyServiceRpc {
+    shinami_key_createSession(secret: string): string;
+}
+const keyService = rpcClient<KeyServiceRpc>(KEY_SERVICE_RPC_URL);
+  
+// Wallet Service interaction setup
+interface WalletServiceRpc {
+    shinami_wal_createWallet(walletId: string, sessionToken: string): string;
+    shinami_wal_getWallet(walletId: string): string;
+    shinami_wal_signTransactionBlock(walletId: string, sessionToken: string, txBytes: string): string;
+    shinami_wal_executeGaslessTransactionBlock(
+        walletId: string, 
+        sessionToken: string, 
+        txBytes: string, 
+        gasBudget: number, 
+        options?: {}, 
```

Comment:
Can probably use `SuiTransactionBlockResponseOptions`.

Reply:
I'm actually leave this as `{}` for now. The sui js library is at odds (missing `showRawInput`) with the version of `SuiTransactionBlockResponseOptions` in the rust code, so we get errors if `showRawInput` is omitted from the request.